tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -509,14 +539,21 @@ describe('Creating And Updating Ranked Matches', () => {
     cy.loginPlayer(playerOne.username, playerOne.password);
     cy.setupGameAsP0(true, true);
   });
-  it('Creates a match when two players play a ranked game for the first time this week', function () {
+  it.only('Creates a match when two players play a ranked game for the first time this week', function () {
```

Comment:
Remove this .only here.

Reply:
thanks for catching this! I always forget to remove.only.

Reply:
I also always forget them, lol. Got @itsalaidbacklife working overtime catching them. Nice to switch it up and catch someone else for a change.

Reply:
Yeah I would love to work out a linting rule for this. No `.only(` in .spec files